h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -822,6 +822,7 @@ public void clearInitState() {
   protected transient String[] _origNames; // only set if ModelBuilder.encodeFrameCategoricals() changes the training frame
   protected transient String[][] _origDomains; // only set if ModelBuilder.encodeFrameCategoricals() changes the training frame
 
+  public boolean _checkConstResponse(){return _parms._check_constant_response;}
```

Comment:
please get rid off this method, just use the field directly just like here

    if (expensive && error_count() == 0 && _parms._auto_rebalance) {